include/zephyr/drivers/fuel_gauge.h

```diff
@@ -82,25 +78,16 @@ enum fuel_gauge_property {
 	FUEL_GAUGE_DESIGN_CAPACITY,
 	/** Design Voltage (mV) */
 	FUEL_GAUGE_DESIGN_VOLTAGE,
-	/** AtRate (mA or 10 mW) */
-	FUEL_GAUGE_SBS_ATRATE,
-	/** AtRateTimeToFull (minutes) */
-	FUEL_GAUGE_SBS_ATRATE_TIME_TO_FULL,
-	/** AtRateTimeToEmpty (minutes) */
-	FUEL_GAUGE_SBS_ATRATE_TIME_TO_EMPTY,
-	/** AtRateOK (boolean) */
-	FUEL_GAUGE_SBS_ATRATE_OK,
-	/** Remaining Capacity Alarm (mAh or 10mWh) */
-	FUEL_GAUGE_SBS_REMAINING_CAPACITY_ALARM,
-	/** Remaining Time Alarm (minutes) */
-	FUEL_GAUGE_SBS_REMAINING_TIME_ALARM,
 	/** Manufacturer of pack (1 byte length + 20 bytes data) */
 	FUEL_GAUGE_MANUFACTURER_NAME,
 	/** Name of pack (1 byte length + 20 bytes data) */
 	FUEL_GAUGE_DEVICE_NAME,
 	/** Chemistry (1 byte length + 4 bytes data) */
 	FUEL_GAUGE_DEVICE_CHEMISTRY,
 
+	/** Reserved for start of SBS specific properties. See the fuel_gauge_sbs.h header */
+	_FUEL_GAUGE_SBS_START,
```

Comment:
Might be nice to follow how sensor.h does this, e.g. 

FUEL_GAUGE_COMMON_COUNT,

FUEL_GAUGE_PRIV_START = FUEL_GAUGE_COMMON_COUNT,

FUEL_GAUGE_MAX = INT16_MAX,